contracts/src/core/EigenDAServiceManager.sol

```diff
@@ -64,13 +76,15 @@ contract EigenDAServiceManager is Initializable, OwnableUpgradeable, EigenDAServ
 
     function initialize(
         IPauserRegistry _pauserRegistry,
-        address initialOwner
+        address _initialOwner,
+        address _batchConfirmer
     )
         public
         initializer
     {
         _initializePauser(_pauserRegistry, UNPAUSE_ALL);
-        _transferOwnership(initialOwner);
+        _transferOwnership(_initialOwner);
+        batchConfirmer = _batchConfirmer;
```

Comment:
event?

Reply:
should we make this immutable for gas savings?

Reply:
think its better if mutable incase of disperser failure